script.js

```diff
@@ -107,25 +110,37 @@ const applyToCells = callback => {
   }
 };
 
+//Function to implement the Color Selector
+let colorSelector = document.getElementById("color_choose"), color_change;
```

Comment:
This is a bit confusing. It's the equivalent of 
```
let colorSelector, color_change;
colorSelector = document.getElementById("color_choose");
```
But the one liner is much harder to read. Usually it's preferred to use `const` for non changing values.

Reply:
What is the default value of `color_change`? `colorSelect` is fired on an onchange event. You should either manually set the value to the default color in the selector, or better yet use the value of the element you get, `colorSelector`.